ui/src/app/pages/data/concept/concept-search.tsx

```diff
@@ -303,10 +311,13 @@ export const ConceptSearch = fp.flow(withCurrentCohortSearchContext(), withCurre
                           {errors && <ValidationError>
                             {summarizeErrors( errors && errors.editName)}
                           </ValidationError>}
-                          <TextArea value={editDescription} disabled={editSaving}
-                                    style={{marginBottom: '0.5rem'}} data-test-id='edit-description'
-                                    onChange={v => this.setState({editDescription: v})}/>
-                          <div style={{marginBottom: '0.5rem'}}>
+                          <TextAreaWithLengthValidationMessage initialText={editDescription}
+                                                               id='edit-description'
+                                                               textBoxStyleOverrides={{width: '100%'}}
+                                                               maxCharacters={1000}
+                                                               tooLongWarningCharacters={950}
+                                                               onChange={v => this.setState({editDescription: v})}/>
+                          <div style={{margin: '0.5rem 0'}}>
                             <Button type='primary' style={{marginRight: '0.5rem'}}
```

Comment:
Since we have 2 errors (name cannot be empty and description too long ) it will be nice to have a tooltip to the button

Reply:
Good idea, done.
<img width="634" alt="Screen Shot 2021-03-04 at 1 59 47 PM" src="https://user-images.githubusercontent.com/40036095/110023012-3fd84500-7cf2-11eb-9758-bacd090683a3.png">